mappings/remediation_advice.json

```diff
@@ -338,7 +338,7 @@
           "children": [
             {
               "id": "lack_of_forward_secrecy",
-              "remediation_advice": "As a best practice, consider using Forward Secrecy to help prevent the compromise of future sessions and limit compromise only to past sessions.",
+              "remediation_advice": "As a best practice, consider using Forward Secrecy. This ensures that broad decrypting of recorded HTTPS sessiosn can not happen, should the private key become publicly accessable.",
```

Comment:
1. Typo: "sessiosn"
2. Typo: "accessable"
3. Missing period at the end (don't judge me;))

Reply:
Missing a period does mean the sentence isn't finished. So then it's wrong.